Author: George Kiagiadakis <kiagiadakis.george@gmail.com>
Description: Sets compiler flags to hide non-exported symbols from the ABI.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -22,6 +22,8 @@
 set(GENERIC_LIB_SOVERSION "1")
 set(VERSION ${GENERIC_LIB_VERSION})
 
+set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fvisibility=hidden -fvisibility-inlines-hidden")
+
 add_subdirectory(bluedevil)
 
 option(LIBBLUEDEVIL_BUILD_API_DOCS "Build libbluedevil API documentation")
